### ISSUE TITLE
Add default to define a default context for each endpoint

### ISSUE THREAD

**Issue by roberthoner** [payout/rester#39]
This will be useful for use when in the development environment (not while running tests).

Example:
```yml
/v1/accounts:
  POST:
    With expected response:
      default: true
      response:
        code: 201
        body:
          account_id: 1234
```

**Comment by roberthoner** [comment]
This is no longer needed after #44. Now we select the first context that matches the request data.
